CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+
+#### unreleased - upcoming
+
+- Add `requireUserWithAnyRole` and `anyRoleExcept`, to be used in the
```

Comment:
currently, if an user is both an analyst and an operator, they will see the calls list, as this is the way `anyRoleExcept` works. The problem with specifying the roles that _can_ see the list, is that when a new role is added you need to update all the parts in the code where a list of roles that can do something was used